api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java

```diff
@@ -1380,182 +1400,222 @@ public CohortDefinition getBreastfeedingPatientsStartingART() {
    *
    * @return
    */
-  /* public CohortDefinition getPatientsWhoAreActiveAndParticipateInDsdModel() {
```

Comment:
Please undo this commenting style change, it makes it harder to review the pull request. Same for the other methods.

Reply:
done